utils/telegraf/Makefile

```diff
@@ -76,7 +70,7 @@ define Package/telegraf/install
 	$(call GoPackage/Package/Install/Bin,$(1))
 	$(INSTALL_DIR) $(1)/etc/init.d $(1)/etc/config
 	$(INSTALL_BIN) ./files/etc/init.d/telegraf $(1)/etc/init.d/telegraf
-	$(INSTALL_CONF) $(PKG_BUILD_DIR)/etc/telegraf.conf $(1)/etc/telegraf.conf
+	$(1)/usr/bin/telegraf config > $(1)/etc/telegraf.conf
```

Comment:
`telegraf` here is a program cross-compiled for the target device. It almost certainly will not run on the build system.

Reply:
Right, this was my unsuccessful experiment to generate telegraf.conf file facing my limited knowledge of OpenWrt Makefile system.

Reply:
I suggest adding a [uci-defaults](https://openwrt.org/docs/guide-developer/uci-defaults) script that (after testing if the config file already exists) generates the config file. This script will be run on package install or on first image boot (if the package was integrated into a custom image).

A [postinst](https://openwrt.org/docs/guide-developer/packages#packagepostinst) script is usually another option but it gets run when a custom image is created, which is still on the build system (not on the target).